packages/manager/src/features/CloudPulse/Alerts/constants.ts

```diff
@@ -272,3 +279,6 @@ export const entityLabelMap = {
 
 export const entityTypeTooltipText =
   'Select a firewall entity type to filter the list in the Entities section. The metrics and dimensions in the Criteria section will update automatically based on your selection.';
+
+export const NotificationChannelTableAlertTooltipText =
```

Comment:
Keep it short, we are already in context of notification channels
```suggestion
export const ChannelAlertsTooltipText =
```